docs/content/SRS/SRS.md

```diff
@@ -163,33 +231,83 @@ _The following Software Requirements Specification for `tinymorph` is using [Vol
 
 ### 10.1 Appearance Requirements
 
-[Insert your content here.]
+LF-A1: The Tinymorph application shall adopt a unified and sophisticated visual design.
+
+Rationale: A uniform and elegant design ensures brand recognition and provides a visually pleasing experience for users, fostering greater engagement and trust in the platform.
+
+LF-A2. The application must utilize a consistent typography system across all user interfaces.
+
+Rationale: Maintaining consistency in typography ensures readability and prevents user distraction or confusion, contributing to a cohesive user experience.
 
 ### 10.2 Style Requirements
 
-[Insert your content here.]
+LF-S1. The design of the application will be minimalist, utilizing clean lines and a limited color palette.
```

Comment:
Maybe we can mention a 16-bit or monotonic colour pallete to avoid distraction?

Reply:
Discussed using monotonic color palette under the style requirements(LF-S1).